service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/ConditionMapper.java

```diff
@@ -3,37 +3,67 @@
 import static org.hl7.fhir.dstu3.model.IdType.newRandomUuid;
 
 import java.util.Date;
+import java.util.Optional;
 
+import org.apache.commons.lang3.StringUtils;
 import org.hl7.fhir.dstu3.model.CodeableConcept;
+import org.hl7.fhir.dstu3.model.Coding;
 import org.hl7.fhir.dstu3.model.Condition;
 import org.hl7.fhir.dstu3.model.Encounter;
 import org.hl7.fhir.dstu3.model.Reference;
 import org.springframework.stereotype.Component;
 
 import lombok.AllArgsConstructor;
+import uk.nhs.adaptors.oneoneone.cda.report.enums.Language;
 import uk.nhs.adaptors.oneoneone.cda.report.util.DateUtil;
 import uk.nhs.adaptors.oneoneone.cda.report.util.NodeUtil;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Component5;
 import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Encounter;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Section;
 
 @Component
 @AllArgsConstructor
 public class ConditionMapper {
 
     private final NodeUtil nodeUtil;
 
-    public Condition mapCondition(POCDMT000002UK01Encounter itkEncounter, Encounter encounter) {
+    public Condition mapCondition(POCDMT000002UK01Section section, POCDMT000002UK01Encounter itkEncounter, Encounter encounter) {
         Condition condition = new Condition();
 
         condition.setIdElement(newRandomUuid());
 
         Date assertedDate = DateUtil.parse(itkEncounter.getEffectiveTime().getValue());
 
+        for (POCDMT000002UK01Component5 component : section.getComponentArray()) {
+            if (component.getSection() != null) {
+                if (component.getSection().isSetLanguageCode()) {
+                    if (component.getSection().getLanguageCode().isSetCode()) {
+                        Optional<Language> language = Language.fromCode(component.getSection().getLanguageCode().getCode());
+                        language.ifPresent(lang -> {
+                            Coding coding = new Coding();
+                            if (!StringUtils.isBlank(language.get().getDisplay())) {
```

Comment:
Don't use `language.get()`. You already have the optional unwrapped in `lang` variable.